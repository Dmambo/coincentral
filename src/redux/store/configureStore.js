import { configureStore } from '@reduxjs/toolkit';
import cryptoReducer from '../reducer/crypto';

const store = configureStore({
  reducer: {
    crypto: cryptoReducer,
  },
});

export default store;
