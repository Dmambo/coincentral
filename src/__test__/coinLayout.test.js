import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import CoinLayout from '../components/coinLayout/coinLayout';
import store from '../redux/store/configureStore';

// create a test for the components with snapshots

test('renders CoinLayout component', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <CoinLayout />
    </Provider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
