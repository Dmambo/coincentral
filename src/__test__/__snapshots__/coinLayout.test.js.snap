// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CoinLayout component 1`] = `
<DocumentFragment>
  <div
    class="coin-app"
  >
    <h2
      class="allCoins"
    >
      <div
        class="coin-search"
      >
        <h4
          class="coin-text"
        >
          Search
        </h4>
        <form>
          <input
            class="coin-input"
            placeholder="Search"
            type="text"
            value=""
          />
        </form>
      </div>
    </h2>
    <div
      class="grid-coin"
    >
      <p>
        No coins found
      </p>
    </div>
    <button
      class="showBtn"
      type="button"
    >
      Show More
    </button>
  </div>
</DocumentFragment>
`;
